client/X11/xf_keyboard.c

```diff
@@ -191,6 +194,28 @@ void xf_keyboard_send_key(xfContext* xfc, BOOL down, BYTE keycode)
 	DWORD rdp_scancode;
 	rdpInput* input;
 	input = xfc->context.input;
+
+	if (keycode == 0x25)
+	{
+		if (down)
+			controlDown = TRUE;
+		else
+			controlDown = FALSE;
+	}
+
+	if (keycode == 0x40)
+	{
+		if (down)
+			laltDown = TRUE;
+		else
+			laltDown = FALSE;
+	}
+
+	if (laltDown && controlDown && keycode == 0x73 && down == 1)
```

Comment:
Better check against `rdp_scancode` and the `RDP_SCANCODE*` macros instead of these magic numbers